src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -89,31 +88,16 @@ export const ConfirmBidRoute: React.FC<ConfirmBidProps> = props => {
               max_bid_amount_cents: maxBidAmountCents,
             },
           },
-        }
-      )
-    })
+        })
+      }
+    )
   }
 
-  function handleMutationError(
-    actions: FormikActions<object>,
-    error: Error,
-    bidderId: string
-  ) {
+  function onJsError(actions: BidFormActions, error: Error, bidderId: string) {
     logger.error(error)
-
-    let errorMessages: string[]
-    if (Array.isArray(error)) {
-      errorMessages = error.map(e => e.message)
-    } else if (typeof error === "string") {
-      errorMessages = [error]
-    } else if (error.message) {
-      errorMessages = [error.message]
-    }
-
-    trackConfirmBidFailed(bidderId, errorMessages)
-
+    trackConfirmBidFailed(bidderId, [error.message])
     actions.setSubmitting(false)
-    actions.setStatus("submissionFailed")
+    // actions.setStatus("submission Failed.")
```

Comment:
Maybe this should be deleted/not sure if it is still necessary.

Reply:
It seems like this is how Formik keeps track of error state, so I brought it back and used it to show an error message.